docs/reference/how-to/knn-search.asciidoc

```diff
@@ -66,6 +71,14 @@ reduction techniques like PCA. When experimenting with different approaches,
 it's important to measure the impact on relevance to ensure the search
 quality is still acceptable.
 
+[discrete]
+=== Reduce vector memory foot-print
+
+As of 8.12 the default <<dense-vector-element-type,`element_type`>> is `float`. But this can be
```

Comment:
The doc is per version so not sure if it's worth mentioning `8.12`?

Reply:
DOH! for sure, docs are already versioned!